### ISSUE TITLE
Language  attribute in <shortDescription>

### ISSUE THREAD

**Issue by AlexErk** [ebu/radiodns-manager#134]
![image](https://user-images.githubusercontent.com/4168960/57772767-9d169700-7716-11e9-94fd-a309e3992fb0.png)

While language in Web GUI is set to german, the language in exported SI.xml is set to "fr" for  the `<shortDescription>` in `<service>`
e.g.:
`<shortDescription xml:lang="fr">B1 Obb</shortDescription>`

**Comment by magicbadger** [comment]
When no language is set on the ServiceProvider, both the default document language and the service provider language and the service language are given as `None`:

```
<serviceInformation xmlns="http://www.worlddab.org/schemas/spi/31" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.worlddab.org/schemas/spi/31 spi_31.xsd" creationTime="2019-05-13T05:44:04" originator="EBU.io RadioDNS Services v3" xml:lang="None">
```
```
<shortDescription xml:lang="None">European Broadcasting Union</shortDescription>
```
```
<link url="http://www.ebu.ch" mimeValue="text/html" description="EBU Webpage" xml:lang="None"/>
```
```
<link url="http://www.ebu.io" mimeValue="text/html" xml:lang="None"/>
```


However, the service language seems to be stuck on `fr`.

```
```

**Comment by magicbadger** [comment]
closed in [release 4.1.1](https://github.com/ebu/radiodns-manager/releases/tag/4.1.1)
